XML/XML.sublime-syntax

```diff
@@ -25,7 +25,7 @@ variables:
 
 contexts:
   main:
-    - match: '(<\?)\s*({{qualified_name}})'
+    - match: '(<\?)(xml)(?=\s)'
```

Comment:
Should this also match mixed case `xml`?

It seems with these changes, this will only match lowercase, and the new match won't match any capitalization of `xml`, leaving `XML` matched by nothing. Is that correct?